README.md

```diff
@@ -1,23 +1,52 @@
 # FlashcardsXBlock
+
 An Open edX platform XBlock to create and display flashcards.
 
-The flashcards are added like this:
+## Installation
+
+In a Tutor deployment, you can add it to `OPENEDX_EXTRA_PIP_REQUIREMENTS`, installing from git:
+
+```yaml
+OPENEDX_EXTRA_PIP_REQUIREMENTS:
+- git+https://github.com/open-craft/FlashcardsXBlock.git@main
+```
+
+Or from a [Pypi release](https://pypi.org/project/FlashcardsXBlock/):
 
-```html
-<flashcards title="Capital cities">
-<flashcard front="Croatia" back="Zagreb" />
-<flashcard front="France" back="Paris" />
-</flashcards>
+```yaml
+OPENEDX_EXTRA_PIP_REQUIREMENTS:
+- FlashcardsXBlock==1.0.1
 ```
 
-The XBlock in Studio looks like this:
-![Flashcard_edit](flashcardsxblock_edit.png)
+## Usage
+
+Once the Flashcards XBlock is installed in the Open edX environment,
+you can navigate to the "Advanced settings" of the course in Studio or Authoring,
+then add "flashcards" to the "Advanced module list".
+
+Then navigate to the unit in the course where you want to add a Flashcards component,
+and select "Flashcards" from the "Advanced" component list.
+
+## Screenshots
 
-Output looks like this:
+Student view:
 
-![Flashcard](flashcardsxblock.png)
+![student view: starting screen](./screenshots/view-1.png)
+
+![student view: a flashcard](./screenshots/view-2.png)
+
+Editing modal in Authoring:
+
+![edit view: edit screen one, where you can select theming](./screenshots/edit-1.png)
+
+![edit view: edit screen two, where you can enter flashcard content](./screenshots/edit-2.png)
+
+## Development notes
```

Comment:
We can a small section about steps to build and develop frontend, since this is a react heavy xblock it might be good to have.